app-emacs/emms/emms-20.2-r1.ebuild

```diff
@@ -23,7 +23,7 @@ LICENSE="GPL-3+ FDL-1.1+"
 SLOT="0"
 
 RDEPEND="
-	media-libs/taglib
+	media-libs/taglib:=
 "
 BDEPEND="
 	${RDEPEND}
```

Comment:
@xgqt this can't be right for sure? Seemingly introduced in ce47c9138c8110850e449b05a024848944b3c00d

Reply:
What is your issue here @a17r? The only bug I see here is that it should be DEPEND instead of BDEPEND

Reply:
that's exactly what I'm talking about, I'm simply asking since this was not introduced as an EAPI bump (was already EAPI-8 before) but simply changed between version bumps.

Reply:
Must have been a spelling mistake. The correct value is DEPEND. If this PR can correct it then it would be nice.

Reply:
In progress.